src/interfaces/IEthMultiVault.sol

```diff
@@ -50,9 +51,9 @@ interface IEthMultiVault {
         /// @dev fee paid to the protocol when depositing vault shares for the triple vault upon creation
         uint256 tripleCreationProtocolFee;
         /// @dev static fee going towards increasing the amount of assets in the underlying atom vaults
-        uint256 atomDepositFractionOnTripleCreation;
+        uint256 totalAtomDepositsOnTripleCreation;
```

Comment:
3.11 - Misleading Variable Name

We have renamed quite a few instances of variables pertaining to the amounts that get deposited in constituent atoms from triple deposits / creations.  Since there are a few hundred instances of these variables across the repo, we will not tag them all with comments -- but they should be clearly visible across the diff.